contracts/src/v0.8/interfaces/ChainlinkPoRAddressList.sol

```diff
@@ -0,0 +1,6 @@
+pragma solidity ^0.8.0;
+
+interface ChainlinkPoRAddressList {
```

Comment:
Do we care about the prefix here?

```suggestion
interface PoRAddressList {
```

Reply:
Nope.

Reply:
At least not in the interface name. I thought it would help avoid collisions for the method names, but I don't feel too strongly about that.